marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/mapping/getUrisAndDoc/suiteSetup.sjs

```diff
@@ -0,0 +1,32 @@
+declareUpdate();
```

Comment:
I think you can reuse the test-data/content concept here - the docs are written to "raw-content" by default, so you can use that as your collection to query on instead of "finalCollection".